crates/ruff_linter/src/rules/mccabe/rules/function_is_too_complex.rs

```diff
@@ -98,6 +98,7 @@ fn get_complexity_number(stmts: &[Stmt]) -> usize {
             Stmt::Match(ast::StmtMatch { cases, .. }) => {
                 complexity += 1;
                 for case in cases {
+                    complexity += 1;
                     complexity += get_complexity_number(&case.body);
                 }
```

Comment:
Similar to the `too-many-branches`, the `match` statement itself doesn't add the complexity but it's the `case` blocks which does.
```suggestion
                for case in cases {
                    complexity += 1;
                    complexity += get_complexity_number(&case.body);
                }
```